FSharpTests.Directory.Build.targets

```diff
@@ -33,6 +33,7 @@
               WarningLevel="$(WarningLevel)"
               WarningsAsErrors="$(WarningsAsErrors)">
           <Output TaskParameter="CommandLineArgs" ItemName="FsiCommandLineArgs" />
+          <Output TaskParameter="TextOutput" ItemName="FsiTextOutput" />
```

Comment:
This now becomes public API of the build task.
What is the supposed task with it?

Reply:
Good point. This made me think and I just did a search. The enclosing target here "RunFSharpScript" is literally used only by FSharpSuite.Tests to run F# scripts via msbuild (!). The modification I did here was to somehow obtain the output of scripts run in such a way.
What's more, the actual "Fsi" task does not seem to have any other usages than this one here. I.e. it is only used for testing.